docs/rules/object-key-newlines.md

```diff
@@ -0,0 +1,68 @@
+# Enforce placing object keys on separate lines (object-key-newlines)
+
+While formatting preferences are very personal, a number of style guides require that
```

Comment:
Good question! I went with "key" originally based on @nzakas' [suggestion](https://github.com/eslint/eslint/issues/5667#issuecomment-210594660) for the rule identifier.
Perhaps we should open it to discussion!